packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -1699,7 +1724,7 @@ export class EditableGrid extends PureComponent<EditableGridProps, EditableGridS
         );
 
         if (showAsTab) {
-            const bulkDisabled = selected.size === 0;
+            const bulkDisabled = selected.size === 0 && !hideCheckboxCol;
```

Comment:
doesn't the constructor here use the `hideCheckboxCol` prop to set the selected state? if so, this line seems like it wouldn't need to change.